commands/publish_dafsa.py

```diff
@@ -0,0 +1,100 @@
+import requests
+from requests.exceptions import HTTPError
+import tempfile
+import subprocess
+from kinto_http import Client, KintoException
+import sys
+
+COMMIT_HASH_URL = "https://api.github.com/repos/publicsuffix/list/commits/master"
```

Comment:
Since we download `https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat`, shouldn't we look at the commits of `public_suffix_list.dat` ? It would be excellent to avoid republishing the PSL if just the README has changed..

Reply:
Yes you're right, will look into this as well.